### PR TITLE
OKTA-113081

### PR DESCRIPTION
**Pull request by larsjohansen-okta**
- Added oauth2.json postman collection

Primary reviewer: @danielzeller-okta 
Secondary reviewer: @federations-okta @mystiberry-okta

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
Once this is added to the collection, Platform Doc will register it with postman and add a "Run in Postman" Here's the JIRA for that task: OKTA-114361

File: _source/_docs/api/postman/oauth2.json [hunk 1]
```diff
@@ -0,0 +1,2280 @@
...
+			"name": "Get OAuth2 Claim",
+			"description": "",
+			"collectionId": "232d9502-3dd1-aa0d-9116-2bb7af8d80fe",
+			"responses": [],
+			"rawModeData": "{\n\t\"defaultResourceUri\": \"https://api.resource.com\",\n\t\"description\": \"Authorization Server Description\",\n\t\"name\": \"Authorization Server\"\n}"
+		},
+		{
+			"id": "438732d8-4a80-b382-83a7-c1fcf46fd070",
+			"headers": "Accept: application/json\nContent-Type: application/json\nAuthorization: SSWS {{apikey}}\n",
+			"url": "{{url}}/api/v1/as/{{authorizationServerId}}/resources/{{resourceId}}/policies?type=OAUTH_AUTHORIZATION_POLICY",
```
Comment (danielzeller-okta) [inline _source/_docs/api/postman/oauth2.json]: Why did you include the query param? We don't take any on this endpoint.

File: _source/_docs/api/postman/oauth2.json [hunk 2]
```diff
@@ -0,0 +1,2280 @@
...
+			"dataMode": "raw",
+			"tests": "",
+			"currentHelper": "normal",
+			"helperAttributes": {},
+			"time": 1479040727361,
+			"name": "Update OAuth2 Scope",
+			"description": "",
+			"collectionId": "232d9502-3dd1-aa0d-9116-2bb7af8d80fe",
+			"responses": [],
+			"rawModeData": "{\n\t\"description\": \"Drink beer\",\n\t\"name\": \"beer:drink\"\n}"
```
Comment (danielzeller-okta) [inline _source/_docs/api/postman/oauth2.json]: I'm wondering if we want to be putting this in our external-facing docs? I personally don't think it's a big deal, but we've already had to remove the glass of beer security image from the product.
@mystiberry-okta / @cwu-okta can you give a 👍  or 👎 ?

Comment (mystiberry-okta) [inline _source/_docs/api/postman/oauth2.json]: I'll create a JIRA to change Drink Beer to something better. Don't want to hold up this work for that.

**Pull request merged**